atom/install

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
+
+set -e
+
+prefix=$(cat prefix)
+
+./script/build --install="${prefix}"
+for icon in resources/app-icons/stable/png/*.png
+do
+    size=$(basename "${icon}" | cut -d. -f1)
+    install -D "${icon}" "${prefix}/share/icons/hicolor/${size}x${size}/apps/io.atom.Atom.png"
```

Comment:
Have to check it, but I'm not exactly renaming here. All the icons have the name like `<size>.png`, so `128.png` or `64.png`. But I'll check this `rename-icon` thing, thanks.